package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "integration-nameOfYourIntegration",
-  "version": "1.0.3",
+  "name": "integration-UserLeap",
```

Comment:
Do you currently have any other npm packages that are scoped to your company name?  if so, you should scope this to your userleap namespace, ie @userleap/mparticle-web-kit or similar. this way for people who install it via npm it makes more sense rather than this generic name here.  
once it's up on npm, we include it here in our docs page. you'll note that most of these are @mparticle/ but since you can't scope it to us since you maintain it, you'll have to use your namespace or other.  branch uses `branch-mparticle-web-kit` as an example.  it's up to you how you want to name the npm package.